indexing-service/src/test/java/org/apache/druid/indexing/seekablestream/SeekableStreamSupervisorSpecTest.java

```diff
@@ -131,8 +135,21 @@ public void setUp()
     monitorSchedulerConfig = EasyMock.mock(DruidMonitorSchedulerConfig.class);
     supervisorStateManagerConfig = EasyMock.mock(SupervisorStateManagerConfig.class);
     supervisor4 = EasyMock.mock(SeekableStreamSupervisor.class);
+    lagStats = EasyMock.mock(LagStats.class);
```

Comment:
`LagStats` is a simple POJO. We shouldn't have to mock it.

Reply:
I need to spy that class so as to confirm that the config override is working and not calling SUM when MAX is specified and viceversa.